components/dropdown/dropdown-content-styles.js

```diff
@@ -30,17 +35,30 @@ export const dropdownContentStyles = css`
 		opacity: 0.9;
 	}
 
-	:host([opened]) {
-		-webkit-animation: var(--d2l-dropdown-animation-name) 300ms ease;
```

Comment:
It looks like there is some other `-webkit-` stuff in here.  Although not required for your changes, I think it would be good to clean that up since we are excluding them for the new styles being added.